### DISCUSSION TITLE
Mirai 生态概览里思维导图都不见了

### DISCUSSION THREAD

**Discussion by aimeini** [mamoe/mirai#911] [Q&A] [answered]
贴个图片也行啊

**Comment by sandtechnology** [comment]
https://github.com/mamoe/mirai/blob/dev/docs/mirai-ecology.md

**Comment by mzdluo123** [comment]
也有可能是因为国内网络的原因，图片无法正常显示，请自行解决

**Comment by StageGuard** [comment, marked as answer]
生态概览中的图片为 Mermaid 实时生成，并非静态图片链接。
若图片长时间未加载出来，请检查国际网络连接是否通畅或尝试刷新。
